examples/github-appcards/netlify/functions/project-cards.js

```diff
@@ -0,0 +1,168 @@
+/* 
+
+  This endpoint updates Miro App Card statuses when there is an update to their poisition
+  or column in GitHub (i.e. from "In Progress" to "Done")
+
+**/
+import { createClient } from "@supabase/supabase-js";
+import fetch from "node-fetch";
+
+const supabase = createClient(
+  process.env.VITE_DATABASE_URL,
+  process.env.VITE_DATABASE_PUBLIC_KEY
+);
+
+exports.handler = async function (event) {
+  if (!event.body) {
+    return {
+      statusCode: 200,
+      body: JSON.stringify({ message: "No Body Found" }),
+    };
+  }
+
+  // Get GitHub project card & Issue Number
+  const body = JSON.parse(event.body);
+  const gitHubProjectCard = body.gitHubProjectCard;
+  const gitHubProjectColumnId = gitHubProjectCard.column_id;
+  const gitHubIssueNumer = gitHubProjectCard.content_url
+    .split("https: //api.github.com/repos/addisonschultz/github-cards/issues/")
+    .pop();
+
+  const headers = {
+    Accept: "application/vnd.github.v3+json",
+    Authorization: `token ${process.env.VITE_GH_ACCESS_TOKEN}`,
+  };
+
+  const gitHubProjectColumn = await fetch(
+    `https://api.github.com/projects/columns/${gitHubProjectColumnId}`,
+    {
+      method: "GET",
+      headers: headers,
+    }
+  )
+    .then((response) => response.json())
+    .then((result) => {
+      return result;
+    })
+    .catch((error) => console.error(error));
+
+  // Get column name
+  const gitHubProjectColumnName = gitHubProjectColumn.name;
+
+  // Get relevant cards associated with this issue/project card
+  const { data, error } = await supabase
+    .from("card-mapping")
+    .select(
+      "id, miroAppCardId::text, gitHubIssueId, miroUserId::text, gitHubUsername, created_at, miroBoardId, gitHubIssueNumber, auth ( access_token )"
+    )
+    .eq("gitHubIssueNumber", gitHubIssueNumer);
+
+  // No Miro App Card Found
+  if (error) {
+    return {
+      statusCode: 200,
+      body: JSON.stringify({
+        message: "No Miro App Card found for this project card",
```

Comment:
Suggestion:

"This GitHub project card doesn't have a corresponding Miro app card."